templates/whatson/event_page.html

```diff
@@ -150,38 +142,42 @@ <h2 class="tna-heading-l">Need to know</h2>
             {% endif %}
 
             {# Hosts and speakers #}
-            <h2 class="tna-heading-l">Your host{{ page.hosts.all|pluralize }} and speaker{{ page.speakers.all|pluralize }}</h2>
-
-            {% if page.hosts %}
-                {% with plural=page.hosts.all|pluralize %}
-                    {% include "components/people.html" with people=page.hosts title="Host"|add:plural %}
-                {% endwith %}
-            {% endif %}
-
-            {% if page.speakers %}
-                {% with plural=page.speakers.all|pluralize %}
-                    {% include "components/people.html" with people=page.speakers title="Speaker"|add:plural %}
-                {% endwith %}
+            {% if page.hosts.all or page.speakers.all %}
```

Comment:
I think you can just check if `page.hosts` exists here rather than using the `.all` as well. Might be worth double checking which is more efficient - I have a feeling using `.all` is less efficient but need to double check

Reply:
Another option is to use with to cache it:

``` 
{% with page.hosts.all as hosts %}
    {% if hosts %}
    ...
    {% endif %}
{% endwith %}

Reply:
Oh this is really good to know - using `page.hosts` and `page.speakers` wasn't working so I added the `.all`, but using `{% with %}` has fixed that - now updated. Thank you!